src/battle_script_commands.c

```diff
@@ -14465,6 +14465,7 @@ static void Cmd_trywish(void)
             gBattleMoveDamage = max(1, GetNonDynamaxMaxHP(gBattlerAttacker) / 2);
         }
 
+        gHitMarker |= HITMARKER_IGNORE_DISGUISE;
```

Comment:
shouldn't you clear that somewhere as well?

Reply:
It's cleared during `HandleAction_ActionFinished`